metrix-mapping/src/main/java/com/powsybl/metrix/mapping/log/ZeroDistributionKeyInfo.java

```diff
@@ -0,0 +1,27 @@
+package com.powsybl.metrix.mapping.log;
+
+import java.util.List;
+import java.util.Objects;
+
+public class ZeroDistributionKeyInfo extends AbstractLogBuilder implements LogDescriptionBuilder {
+
+    private final String timeSeriesName;
+
+    private final double timeSeriesValue;
+
+    private final List<String> equipmentIds;
+
+    public ZeroDistributionKeyInfo(String timeSeriesName, double timeSeriesValue,
+                                   List<String> equipmentIds) {
+        this.timeSeriesName = Objects.requireNonNull(timeSeriesName);
+        this.timeSeriesValue = timeSeriesValue;
+        this.equipmentIds = Objects.requireNonNull(equipmentIds);
+    }
+
+    public ZeroDistributionKeyInfo build() {
```

Comment:
QCA : RMQ => update return with the new LogContent Class (cf. comment in AbstractLogBuilder)